lib/components/user/places/place-editor.js

```diff
@@ -113,20 +115,32 @@ class PlaceEditor extends Component {
           <FlexFormGroup validationState={errorStates.address}>
             <PlaceLocationField
               className='form-control'
-              inputPlaceholder={isFixed ? `Search for ${place.name} location` : 'Search for location'}
+              inputPlaceholder={intl.formatMessage(
+                {
+                  id: `components.PlaceEditor.${isFixed ? 'locationPlaceholder' : 'genericLocationPlaceholder'}`
+                },
+                {
+                  // Rendered only for fixed locations
+                  placeName: place.name
+                }
+              )}
               location={makeLocationFieldLocation(place)}
               locationType='to'
               onLocationSelected={this._handleLocationChange}
               showClearButton={false}
               static={isMobile()}
             />
             <FormControl.Feedback />
-            {errors.address && <HelpBlock>{errors.address}</HelpBlock>}
+            {errors.address && (
+              <HelpBlock>
+                <FormattedMessage id={errors.address} />
```

Comment:
Please do not use a variable for a message ID. As noted in https://github.com/opentripplanner/otp-react-redux/pull/454/files#r709623526, you could use some kind of identifier to determine the error, but the message ID should still be a hard-coded string.